### ISSUE TITLE
Release 3.2.0 changes the API, should've been 4.0.0

### ISSUE THREAD

**Issue by eemeli** [ocombe/ng2-translate#301]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[ ] feature request
```

**Current behavior**
Release 3.2.0 and in particular commit 481365df changed the signature of `MissingTranslationHandler#handle()`, an exported part of the API. Previously it expected a single `key: string` parameter, now it wants a `params: MissingTranslationHandlerParams`.

**Expected/desired behavior**
According to semver, non-backward compatible changes in the API should increment the major version. This did not happen here, and as dependencies like `^3.1.3` match the `3.2.0` release, having followed the [instructions](../#how-to-handle-missing-translations) is now leading to builds breaking.

**Please tell us about your environment:**

* **ng2-translate version:** 3.2.0
* **Angular version:** 2.1.2
* **Browser:** all
* **Language:** all

**Comment by ocombe** [comment]
Damn you're right, sorry about that, I'll release a new major.
Do you know how to cancel a version on npm?

**Comment by SamVerschueren** [comment]
@ocombe I don't believe that's possible. I think the only way of doing that is checking out the commit right before the breaking change and release a new patch release.

Also, if we're releasing a new major, we might consider removing the `ng2-translate.ts` file in the root of the project. But I leave that to you to decide :).

**Comment by eemeli** [comment]
[`npm unpublish`](https://docs.npmjs.com/cli/unpublish) is the command you're looking for, but that's probably not what you really want. Would recommend reverting 481365d and then re-publishing that as `3.2.1`, and then re-publishing `3.2.0` (possibly with other API changes) as `4.0.0`.

**Comment by ocombe** [comment]
Yes, I deprecated 3.2.0, and released a revert as 3.2.1, it should be ok for you now.
I'll add a few breaking changes and release 4.0.0 today.

Thanks for warning me about that.
